assets/js/push.js

```diff
@@ -122,6 +148,46 @@ jQuery( window ).on( 'load', () => {
 	function distributorMenuEntered() {
 		distributorMenuItem.focus();
 		document.body.classList.toggle( 'distributor-show' );
+
+		if ( distributorPushWrapper.classList.contains( 'loaded' ) ) {
+			return;
+		}
+
+		distributorPushWrapper.classList.add( 'loaded' );
+
+		const data = {
+			action: 'dt_load_connections',
+			loadConnectionsNonce: dt.loadConnectionsNonce,
+			postId: dt.postId
+		};
+
+		const xhr = dt.usexhr ? { withCredentials: true } : false;
+
+		jQuery.ajax( {
+			url: dt.ajaxurl,
+			xhrFields: xhr,
+			method: 'post',
+			data: data
+		} ).done( ( response ) => {
+			setTimeout( () => {
```

Comment:
why are these timeouts required?

Reply:
Don't think they are. Copied this `ajax` function from the other one we have in this file and it uses `setTimeout` (not sure why). But I've removed them from here.